src/shared/utils/channel-utils.ts

```diff
@@ -29,9 +29,11 @@ export const handleChannelAlias = async (
 	if (channelNames.size === 0) {
 		return;
 	}
-	await interaction.deferReply({
-		ephemeral: true,
-	});
+	if (!interaction.replied) {
```

Comment:
Good for now,  but maybe we dont want to handle the interaction replies in the util. Idk what is best.